_pages/documentation/gem5-stdlib/5-local-resources-support.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: documentation
+title: Local Resources Support in gem5
+parent: gem5-standard-library
+doc: gem5 documentation
+permalink: /documentation/gem5-stdlib/local-resources-support
+author: Kunal Pai, Harshil Patel
+---
+
+# Adding a Workload to gem5
+
+This tutorial will walk you through the process of creating a WorkloadResource in gem5 and testing it, through the new gem5 Resources infrastructure introduced in gem5 v23.0.
+
+A workload is set to a board in gem5 through the following line:
+
+``` python
+board.obtain_resource(<ID_OF_WORKLOAD>)
+```
+
+The WorkloadResource with ID '<ID_OF_WORKLOAD>' will be parsed and it will be used to construct the function call it defines. The function call specified in the `"function"` field of the Workload JSON is then executed on the board, along with any parameters it has defined in the `"additional_parameters"` field.
```

Comment:
Can you explain how to find the ID? Maybe a picture of the gem5-resources page with where the ID is?

Reply:
I added one.